app/src/main/java/org/xwiki/android/sync/activities/editcontact/EditContactActivity.java

```diff
@@ -201,17 +242,33 @@ private boolean checkInput() {
         }
     }
 
+    /**
+     * Should be called when a time taking process starts and we want the user
+     * to wait for the process to finish. The UI should gracefully display some
+     * sort of progress bar or animation so that the user knows that the app is
+     * doing some work and has not stalled.
+     *
+     * For example: a network request to the API is made for authenticating
```

Comment:
Javadoc actually being pseudo html it's better to put a <p> between those two paragraphs to actually end up with two paragraphs visually in the end.

Reply:
fixed